nimble/controller/src/ble_ll_sched.c

```diff
@@ -1237,6 +1252,81 @@ ble_ll_sched_next_time(uint32_t *next_event_time)
     return rc;
 }
 
+uint8_t
+ble_ll_sched_get_current_type(void)
+{
+    if ((int32_t)(g_ble_ll_sched_current.end_time - os_cputime_get32()) > 0) {
```

Comment:
What is the meaning of BLE_LL_SCHED_TYPE_NONE? Shouldn't the condition check for <= 0?

Reply:
none means that currently no scheduled item is being executed.
the check above is required since we do not have trigger when scheduled item finishes execution thus `g_ble_ll_sched_current` is not reset automatically - we do this here if its end time is past current time.